### PR TITLE
Fix/32218 responsive visual issues new product page

### LINKED ISSUE #32218: Visual issues on the new product page (mobile)
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

There are a few issues on mobile worth checking.

0. Save and publish display problem.
https://capture.dropbox.com/sNn0AT3eOBRYcCzN

1. Three dots icon is not aligned correctly.
https://capture.dropbox.com/b3owdasT9xJwIskM

2. We could use a little more spacing there.
https://capture.dropbox.com/xugYktCr23hUE0En

3. Features are displayed in a very confusing way.
https://capture.dropbox.com/99WqNlcGRvUq9eev

4. Problems with "Add new file" popup.
https://capture.dropbox.com/ZLJ0vVA1ETZz2eB3

5. Customizable product. Problems with spacing and alignment.
https://capture.dropbox.com/Wwtlef6xog066k7m

6. It is impossible to edit the stock on a mobile device.
https://capture.dropbox.com/dtjVuCfDQeHZ7mUN

7. Alignment problem.
https://capture.dropbox.com/Mw8BgK2GNY9XYWXw

8. Incorrectly displayed prices.
https://capture.dropbox.com/xu2plzoqvpek5GFZ

9. Cost price displayed incorrectly (I'm aware that it's a tricky element to display, maybe a responsive table would be better).
https://capture.dropbox.com/kHk2MsytitYtfaO4

### Expected behavior

_No response_

### Steps to reproduce

Visit the new product page, and places that you see on the screenshots.

### PrestaShop version(s) where the bug happened

8.1.0

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

_No response_

### PR DESCRIPTION
**Pull request by mattgoud**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           |  8.1.x
| Description?      | see #32218 
| Type?             | bug fix
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | go to product page and test each tab (fork : https://github.com/mattgoud/PrestaShop/tree/fix/32218-responsive-visual-issues-new-product-page)
| UI Tests          | Please run UI tests and paste here the link to the run. [Read this page to know why and how to use this tool.](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/ui-tests/).
| Fixed issue or discussion?     | Fixes #32218 (with the exception of point number 4 : creation of a separate issue see #35765
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/pages/product/product_page.scss [hunk 1]
```diff
@@ -2763,6 +2789,53 @@ $mobile-menu-height: 2.6rem !default;
...
+        @include media-breakpoint-up("md") {
+          justify-content: flex-start;
+        }
+
+        h3{
+          margin-bottom: 0;
+        }
+      }
+
+      #product_details_show_condition{
```
Comment (jolelievre) [inline admin-dev/themes/new-theme/scss/pages/product/product_page.scss]:
```suggestion
      #product_details_show_condition {
```

File: admin-dev/themes/new-theme/scss/pages/product/product_page.scss [hunk 2]
```diff
@@ -2763,6 +2789,53 @@ $mobile-menu-height: 2.6rem !default;
...
+        margin-top: 0;
+      }
+
+      .customization-field-switch{
```
Comment (jolelievre) [inline admin-dev/themes/new-theme/scss/pages/product/product_page.scss]:
```suggestion
      .customization-field-switch {
```
Not blocking of course, but I'm surprized the linter doesn't detect this

File: themes/package.json [hunk 3]
```diff
@@ -32,4 +32,4 @@
     "webpack": "^5.37.1",
     "webpack-cli": "^4.7.0"
   }
-}
\ No newline at end of file
+}
```
Comment (jolelievre) [inline themes/package.json]: Not sure this extra EOL should be added

Comment (mattgoud) [reply themes/package.json]: ok @jolelievre, i deleted the extra EOL

**Review by boherm — CHANGES_REQUESTED** [review]
Hi @mattgoud!
LGTM, but you have an issue about one merge commit in your PR, can you recheck? 🙂

**Pull request closed without merging**